src/Microsoft.DotNet.Interactive/ChooseKeyValueStoreKernelDirective.cs

```diff
@@ -18,18 +18,18 @@ public class ChooseKeyValueStoreKernelDirective : ChooseKernelDirective
     {
         NameOption = new Option<string>(
             "--name",
-            "The name of the value to create. You can use #!share to retrieve this value from another subkernel.")
+            Resources.NameOptionDescription)
```

Comment:
This name should probably be more specific. There are other options on other magic commands that also have a name option, so a naming convention that helps disambiguate these would be good to establish.

Reply:
Changed to a more specific name